src/lib/web3-utils.js

```diff
@@ -1,13 +1,15 @@
 import env from '../environment'
+import { providers as Providers } from 'ethers'
 import { solidityKeccak256, id as keccak256 } from 'ethers/utils'
-
+import { InvalidURI, InvalidNetworkType, NoConnection } from '../errors'
 export const soliditySha3 = solidityKeccak256
 export const hash256 = keccak256
 export const DEFAULT_LOCAL_CHAIN = 'private'
 export const ETH_FAKE_ADDRESS = `0x${''.padEnd(40, '0')}`
 
 const ETH_ADDRESS_SPLIT_REGEX = /(0x[a-fA-F0-9]{40}(?:\b|\.|,|\?|!|;))/g
 const ETH_ADDRESS_TEST_REGEX = /(0x[a-fA-F0-9]{40}(?:\b|\.|,|\?|!|;))/g
+const websocketRegex = /^wss?:\/\/.+/
```

Comment:
Actually in this case I think we should be using http or https—we don't use the websockets provider at the moment.